### PR TITLE
Update exercise 4 and rubric

### PR DESCRIPTION
**Pull request by topipa**
This PR updated exercise 4 by removing parts which are found in R demos. New subtasks about computing MCSEs are added. Rubric and markmyassignment tasks are also update

### CODE REVIEW

File: exercises/ex4.Rnw [hunk 1]
```diff
@@ -93,72 +93,85 @@ mark_my_assignment()
...
-\textbf{Hint!} Logarithm of the product of two densities is the sum of the log-densities, i.e. $\log ab = \log a + \log b$.
-
-\item[d)] Plot the posterior density in a grid of points ($\alpha \in [-4,4]$, $\beta \in [-10,30]$) using the {\tt bioassay\_posterior\_density\_plot} function from the {\tt aaltobda} package. Pass the {\tt p\_log\_posterior} function you implemented in c) as argument to {\tt bioassay\_posterior\_density\_plot}.
-\item[e)] Implement two functions in: 1) A function for computing the log importance ratios (log importance weights) for draws from the prior distribution in 1a) when the target distribution is the posterior distribution. 2) A function for exponentiating the log importance ratios and normalizing them to sum to one. Below is a test example, the functions can also be tested with \texttt{markmyassignment}. \textbf{Note!} The values below are \emph{only} a test case, you need to use correct draws for {\tt alpha} and {\tt beta} in the final report. \\
+\end{itemize}
+\item[] {\bf Importance sampling}. Now we discard our posterior draws and
```
Comment (AlejandroCatalina) [inline exercises/ex4.Rnw:122]: is `[]` a typo?

Comment (topipa) [reply exercises/ex4.Rnw]: Not really, it just removed the numbering from enumerate. Without numbering the enumerate is unnecessary, so I removed it

**Comment by MansMeg** [comment]
Great work!

I think only grade on the exact number of digits may be harsh. In the example I think both 2-3 digts would be ok. Maybe be more loose in the rubric here or be very specific what we mean whith the correct number of digits? (At line 109 in ex4.rmd)

Otherwise I think it looks great!

**Comment by topipa** [comment]
> Great work!
> 
> I think only grade on the exact number of digits may be harsh. In the example I think both 2-3 digts would be ok. Maybe be more loose in the rubric here or be very specific what we mean whith the correct number of digits? (At line 109 in ex4.rmd)
> 
> Otherwise I think it looks great!

Requiring exact number of digits is another way of having a separate question "how many digits should be reported based on the MCSE?". The example on line 111 tries to make the requirement as specific as possible. Do you think this should be loosened @avehtari ?

**Pull request merged**